engine/process/main.go

```diff
@@ -98,10 +98,25 @@ func main() {
 
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
+
 	err = cmd.Run()
 
 	if err != nil {
 		print.ProcDebug("error running process: %v\n", err)
-		os.Exit(ERunningProc)
+		if cast, ok := err.(*exec.ExitError); ok {
+			if ws, ok := cast.Sys().(syscall.WaitStatus); ok {
+				if ws.Signaled() {
+					// more verbose err message
+					cmd.Stderr.Write([]byte(err.Error()))
```

Comment:
This returns a value we may want to sanity check and debug log if it fails. It's unlikely to fail, but we can add visibility there just in case.